inst/report/scripts/run_molevolvr_pipeline.R

```diff
@@ -0,0 +1,1296 @@
+  acc2info_out <- fread(input = acc2info, sep = "\t",
+                        header = TRUE, fill = TRUE) %>%
+    mutate(FullAccNum = gsub("\\|", "", FullAccNum)) %>%
+    mutate(FullAccNum = gsub(".*[a-z]", "", FullAccNum))
+
+  # If using web-blast output
+  if (wblast == TRUE) {
+    blast_out <- fread(input = infile_blast, sep = "\t", header = FALSE,
+                       col.names = web_blastp_hit_colnames, fill = TRUE)
+    cleanedup_blast <- blast_out %>%
+      mutate(AccNum = gsub("\\|", "", AccNum)) %>%
+      mutate(AccNum = gsub(".*[a-z]", "", AccNum)) %>%
+      mutate(PcIdentity = round(as.double(PcIdentity), 2))
+
+    # Merge cleaned blast output with acc2info
+    cleanedup_blast <- merge(cleanedup_blast, acc2info_out,
+                             by.x = "AccNum",
+                             by.y = "FullAccNum",
+                             all.x = TRUE)
+    names(cleanedup_blast)[names(cleanedup_blast) == "Species.y"] <- "Species"
+
+    # Additional calculations for PcPositive
+    cleanedup_blast <- cleanedup_blast %>%
+      mutate(PcPosOrig = as.numeric(PcPosOrig)) %>%
+      mutate(AlnLength = as.numeric(AlnLength)) %>%
+      mutate(PcPositive = PcPosOrig)
+
+  } else if (wblast == FALSE) {
+    # If using classic-blast output
+    blast_out <- read_tsv(file = infile_blast, col_names = cl_blast_colnames)
+    cleanedup_blast <- blast_out %>%
+      mutate(AccNum = gsub("\\|", "", AccNum)) %>%
+      mutate(AccNum = gsub(".*[a-z]", "", AccNum)) %>%
+      mutate(Species = gsub(";.*$", "", Species)) %>%
+      mutate(PcIdentity = round(PcIdentity, 2)) %>%
+      mutate(PcPositive = round((PcPosOrig * AlnLength / QLength), digits = 2))
+
+    # Merge cleaned blast output with acc2info
+    cleanedup_blast <- merge(cleanedup_blast, acc2info_out,
+                             by.x = "AccNum",
+                             by.y = "FullAccNum",
+                             all.x = TRUE) %>%
+      select(-Species.x, -TaxID.x)
+    names(cleanedup_blast)[names(cleanedup_blast) == "Species.y"] <- "Species"
+  }
+
+  # TaxID to lineage mapping
+  cleanedup_blast$TaxID <- as.integer(cleanedup_blast$TaxID)
+  lineage_map <- fread("~/awasyn/new_trial/lineage_lookup.txt",
+                       header = TRUE, fill = TRUE,
+                       colClasses = lineage_map_cols)
+
+  # Merge with lineage map and clean up columns
+  mergedLins <- merge(cleanedup_blast, lineage_map, by = "TaxID",
+                      all.x = TRUE) %>%
+    mutate(Species = Species.y, Spp.blast = Species.x) %>%
+    select(any_of(cl_blast_postcln_cols))
+
+  # Add names and prepare the output
+  blast_names <- addName(mergedLins)
+
+  # Create output file name
+  file_name <- file.path(outdir, paste0(prefix, ".blast.cln.tsv"))
+
+  # Write cleaned data to file
+  write_tsv(blast_names, file_name, col_names = TRUE)
+}
+
+
+# Function to run BLASTCLUST on given input
+run_blastclust <- function(infile, suffix, outdir) {
```

Comment:
rename throughout
```suggestion
runBLASTCLUST <- function(infile, suffix, outdir) {
```

Reply:
@awasyn, if this is now running CDHIT, rename to runCDHIT instead of runBLASTCLUST? How drastically different are the two output formats in terms of downstream compatibility? cc: @epbrenner @AbhirupaGhosh